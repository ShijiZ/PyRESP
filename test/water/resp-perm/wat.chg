All values are reported in atomic units
%FLAG TITLE
 molecule 1: water

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
   1  -9.9821230E-33   0.0000000E+00   2.3138460E-01
   2  -2.6101230E-33   1.4941870E+00  -9.2553830E-01
   3  -1.8298510E-16  -1.4941870E+00  -9.2553830E-01

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
   1          8          0    -6.1348640E-01
   2          1          0     3.0674320E-01
   3          1          2     3.0674320E-01

%FLAG PERM DIP LOCAL: 3(I4,X5),I4,X2,E16.7
 dip.no   atm.no   ref.no   ivary      p(opt)
   1        1        2        0     1.3652355E-01
   2        1        3        1     1.3652355E-01
   3        2        1        0    -3.9426391E-02
   4        3        1        3    -3.9426391E-02

%FLAG PERM DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1  -1.3219785E-17   0.0000000E+00  -1.6716413E-01
   2   1.5380606E-34   3.1174039E-02  -2.4137514E-02
   3  -3.8177180E-18  -3.1174039E-02  -2.4137514E-02

%FLAG IND DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1   1.2500949E-17   0.0000000E+00   1.5807445E-01
   2   3.0497441E-18  -1.3764611E-02   4.9221699E-02
   3   4.7354225E-18   1.3764611E-02   4.9221699E-02

