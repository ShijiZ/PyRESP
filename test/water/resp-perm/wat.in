resp-perm for water
 &cntrl
 nmol = 1,
 iqopt = 1,
 ihfree = 1,
 qwt = 0.0005,
 ioutopt = 1,
 ipol = 5,
 igdm = 1,
 exc12 = 0,
 exc13 = 0,
 ipermdip = 1,
 pwt = 0.0005,
 virtual = 0
 &end
1.0
water
    0    3  4
    8    0  0  1
    1    0  0
    1    2  3



