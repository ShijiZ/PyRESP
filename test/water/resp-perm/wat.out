
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
 resp-perm for water
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 1   iqopt       = 1
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00050000
 ipol        = 5   igdm        = 1
 exc12       = 0   exc13       = 0
 ipermdip    = 1   pwt         = 0.00050000
 virtual     = 0

 --------------------
 Molecule Information
 --------------------
 Single-molecule run

 Molecule 1 weight:     1.000 
 Molecule 1 name: water
 Total charge (ich):  0
 Number of centers:  3
 Number of permanent dipoles:  4
    1    8    0    0    1
    2    1    0    0
    3    1    2    3

 -----------------------------
 Atom/Dipole Ivary Information
 -----------------------------
    8    0    0    1
    1    0    0
    1    2    3

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =    3
 Total number of permanent dipoles      =    4
 Weight factor on initial charge restraints=  0.000500
 Weight factor on initial dipole restraints=  0.000500
 There are  1 charge constraints

 Read polarizability and radii information from:
 ../../../polarizability/pGM-pol-2016-09-01
 Assume the units are in a.u.

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =     3
 total number of esp points =   295

 center       X               Y               Z
    1  -9.9821230E-33   0.0000000E+00   2.3138460E-01
    2  -2.6101230E-33   1.4941870E+00  -9.2553830E-01
    3  -1.8298510E-16  -1.4941870E+00  -9.2553830E-01


 ------------------------
 Optimization Information
 ------------------------
 iqopt=1, all q0 and p0 values will be set to 0

 Number of unique UNfrozen charge centers =     2
 Number of unique UNfrozen permanent dipoles =     2

 Non-linear optimization requested.
  qchnge =    2.3560545011E-01
  qchnge =    1.1269091306E-02
  qchnge =    2.9932713936E-03
  qchnge =    1.0361967865E-03
  qchnge =    3.7230045809E-04
  qchnge =    1.3456707478E-04
  qchnge =    4.8705055508E-05
  qchnge =    1.7635431215E-05
  qchnge =    6.3864314852E-06
  qchnge =    2.3128731717E-06
  qchnge =    8.3763168562E-07

 Convergence in   10 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1: water

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         8        0.000000     -0.538212      0       0.000913
    2         1        0.000000      0.269106      0       0.000000
    3         1        0.000000      0.269106      2       0.000000

 Sum over the calculated charges:     -0.000

       Permanent Dipoles Before & After Optimization
 dip.no   element.no   p(init)        p(opt)     ivary    d(rstr)/dp
    1         8        0.000000      0.128291      0       0.003074
    2         8        0.000000      0.128291      1       0.003074
    3         1        0.000000     -0.061886      0       0.000000
    4         1        0.000000     -0.061886      3       0.000000

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          8.5437555E-02
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   2.7563447E-04
 The root-mean-squared error (RMSE = sqrt(RSS/N))               9.6661996E-04
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         5.6799219E-02

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1        1.71713   -0.00000    0.00000   -1.71713

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   -0.83915
        Y    0.00000    1.04245
        Z    0.00000    0.00000   -0.20330

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X    1.04245
        Y    0.00000   -0.20330
        Z    0.00000    0.00000   -0.83915